### PR TITLE
post-game-back-end | Moses Kim

### PR DESCRIPTION
**Pull request by mosesr-kim**
image

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -2,11 +2,47 @@ require('dotenv/config');
...
 
+app.use(jsonMiddleware);
+
 app.use(staticMiddleware);
 
+app.get('/api/games', (req, res, next) => {
+  const sql = `
+  select "gameId",
+         "isJoined",
+         "createdAt"
```
Comment (josephtnguyen) [inline server/index.js:25]: can change this to `select *`

File: server/index.js [hunk 2]
```diff
@@ -2,11 +2,47 @@ require('dotenv/config');
...
+});
 
 const app = express();
 
+app.use(jsonMiddleware);
+
 app.use(staticMiddleware);
 
+app.get('/api/games', (req, res, next) => {
+  const sql = `
```
Comment (josephtnguyen) [inline server/index.js:22]: we only want rows where `isJoined` is `false`

**Review by solomonjin — APPROVED** [review]
💯

**Review by josephtnguyen — APPROVED** [review]
:thinkerguns:

**Pull request merged**